### PR TITLE
Add compound stats

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: rotkehlchen/tests/unit/decoders/test_compound.py [hunk 1]
```diff
@@ -597,3 +598,34 @@ def test_compound_liquidator_side(
...
     ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xC440f3C87DC4B6843CABc413916220D4f4FeD117']])
+def test_compound_liquidation_eth(
+        database: 'DBHandler',
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_accounts,
+) -> None:
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/decoders/test_compound.py:609]: add a small docstring here explainingwhat you test

**Review by LefterisJP — COMMENTED** [review]
All tests passed, but left some comments and you also seem to miss any changes to api.rst

File: rotkehlchen/api/v1/resources.py [hunk 2]
```diff
@@ -1940,6 +1940,28 @@ def get(self, async_query: bool) -> Response:
         return self.rest_api.get_compound_balances(async_query=async_query)
 
 
+class CompoundStatsResource(BaseMethodView):
+    """
+    TODO: @yabirgb: Move this endpoint, aave stats and liquity to a common resource
```
[the remark this answers is not in the document]
Comment (yabirgb) [reply rotkehlchen/api/v1/resources.py:1945]: I believe not that much but I didn't want to add it in this PR

File: rotkehlchen/chain/ethereum/modules/compound/compound.py [hunk 3]
```diff
@@ -184,6 +195,149 @@ def get_balances(
...
+                        f'address: {address} asset: {asset} ',
+                    )
+                else:
+                    usd_price = Inquirer().find_usd_price(asset)
+                    profit_so_far[address][asset] = Balance(
+                        amount=profit_amount,
+                        usd_value=profit_amount * usd_price,
+                    )
+
+            for asset, entry in balance_entry['borrowing'].items():
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/compound/compound.py:287]: You resolved this without adding a short 1 line comment here

**Review by LefterisJP — COMMENTED** [review]
You did not resolve 1 comment and you have not added docs.

**Review by LefterisJP — APPROVED** [review]
Will merge for now, but comments need to be addressed later by @yabirgb

**Pull request merged**